test/unit/DetectChanges/BCBreak/ClassBased/EnumCasesChangedTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RoaveTest\BackwardCompatibility\DetectChanges\BCBreak\ClassBased;
+
+use PHPUnit\Framework\TestCase;
+use Roave\BackwardCompatibility\Change;
+use Roave\BackwardCompatibility\DetectChanges\BCBreak\ClassBased\EnumCasesChanged;
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflector\DefaultReflector;
+use Roave\BetterReflection\SourceLocator\Type\SingleFileSourceLocator;
+use stdClass;
+
+use function array_map;
+use function iterator_to_array;
+
+/** @covers EnumCasesChanged */
```

Comment:
`@covers` always wants a FQCN, sorry @_@ 

```suggestion
/** @covers \Roave\BackwardCompatibility\DetectChanges\BCBreak\ClassBased\EnumCasesChanged */
```

Reply:
Thanks. I probably should have taken the hint from PHPStorm writing it as a FQCN and not offering to shorten it.